.travis.yml

```diff
@@ -1,9 +1,15 @@
 language: java
+# Sudo required for chrome as per https://docs.travis-ci.com/user/chrome
+sudo: required
 
 jdk:
   - openjdk8
   - openjdk11
 
+addons:
+  firefox: latest
+  chrome: stable
```

Comment:
Use specific versions? This might break the builds on updates. (At least for Chrome which requires a specific WebDriver version.)

Reply:
As per irc we'll stick with this for now and see which is more painful.